README.md

```diff
@@ -26,15 +26,26 @@ Our templates offer a rich boilerplate to jump start Go Gin-based application de
   ```
 - Build the binary file.
   ```
-  go build -i -o $GOPATH/bin/nimble-gin github.com/nimblehq/gin-templates
+  go build -o $GOPATH/bin/nimble-gin github.com/nimblehq/gin-templates
   ```
-- Run create the project command.
+- Create the project from a template on a specifying branch.
```

Comment:
Not correct grammatically:

```suggestion
- Create the project from a template on a specific branch.
```

Reply:
Fixed in 2a50e67df6f4112a5236c6eba238a736dacc2df9.
I think if we decide to keeping general use case then specific/specifying should be changed to `main` (which is the default git branch)